### ISSUE TITLE
State change in progress for ..., delaying ...

### ISSUE THREAD

**Issue by macpo** [UncleSamSwiss/ioBroker.loxone#448]
**Describe the bug**  
After I updated Loxone to 14.2 I get messages that a change is delayed or a timeout appears.
I send sensor data to Loxone to display them in the app or use them for the shutters, activate the sun automation for shutters or raise the shutters.
For the sensor data I created virtual inputs.
Today the log is full of the messages that a state change is in progress. The days before there were less then five entries.

I did not notice such a behaviour with the versions 13.x.

**To Reproduce**  
Change virtual inputs in Loxone:
```
on({id: 'zigbee.0.00abcd00054b6a37.temperature', change: "ne"}, function (obj) { setState('loxone.0.abcdefgh-01cc-14d6-ffffc88a6fb2c5e9.value', obj.state.val); });
on({id: 'zigbee.0.00abccdf3c772fe1.illuminance', change: "ne"},  function (obj) { setState('loxone.0.abcdefgh-01d0-559b-ffffc88a6fb2c5e9.value', obj.state.val); });
```

**Expected behaviour**  
There shall be no delaying or timeout of a state change in Loxone, if it is available and running normal.

**Log**
```
2023-07-25 10:49:58.190 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 6839
2023-07-25 10:50:01.443 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 13826
2023-07-25 10:50:06.367 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 17227
2023-07-25 10:50:21.290 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 14309
2023-07-25 10:50:22.506 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 4156
2023-07-25 10:50:31.222 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 12552
2023-07-25 10:50:52.376 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 3668
2023-07-25 10:50:52.954 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 6076
2023-07-25 10:51:22.893 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 5389
2023-07-25 10:51:27.127 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 3234
2023-07-25 10:51:35.917 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 9605
2023-07-25 10:51:37.329 - warn: loxone.0 (942) State change in progress for loxone.0.1826306d-02fc-c390-ffffc88a6fb2c5e9.value, delaying 16.4
2023-07-25 10:51:52.789 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 4691
2023-07-25 10:52:06.904 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 2833
2023-07-25 10:52:17.818 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 4150
2023-07-25 10:52:41.702 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 2489
2023-07-25 10:52:42.622 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 3636
2023-07-25 10:53:02.560 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 3231
2023-07-25 10:53:06.618 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 2204
2023-07-25 10:53:07.176 - warn: loxone.0 (942) Timeout for ack loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value
2023-07-25 10:53:22.538 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 2864
2023-07-25 10:53:25.369 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 7546
2023-07-25 10:53:31.404 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 1912
2023-07-25 10:53:42.390 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 2535
2023-07-25 10:54:27.283 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 2225
2023-07-25 10:54:31.055 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 1697
2023-07-25 10:54:57.173 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 1956
2023-07-25 10:54:59.838 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 5879
2023-07-25 10:55:00.852 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 1482
2023-07-25 10:55:27.084 - warn: loxone.0 (942) State change in progress for loxone.0.1594fd6d-01d0-559b-ffffc88a6fb2c5e9.value, delaying 1738
2023-07-25 10:55:30.675 - warn: loxone.0 (942) State change in progress for loxone.0.1594fde2-009b-edab-ffffc88a6fb2c5e9.value, delaying 1307
2023-07-25 10:55:39.641 - warn: loxone.0 (942) State change in progress for loxone.0.159f892a-03df-ffb6-ffffc88a6fb2c5e9.value, delaying 5206
```

**Versions:**  
 - Adapter version: 3.0.1
 - JS-Controller version: 4.0.24
 - Node version: 16.20.1
 - Operating system: Raspberry Pi OS (Debian Bullseye)
- Miniserver Go Gen. 1

**Comment by raintonr** [comment]
Does this happen periodically? Ie. it things work fine for a while but then the problem appears, but sometimes goes away on it's own?

I have noticed sometimes very slow response to IP communications with MS Gen 1 since the latest update. Rebooting the MS makes things work again for a seemingly random period of time.

When this happens, can you try rebooting your Miniserver and see if the problem goes away, even just for a short amount of time?

**Comment by macpo** [comment]
The mass warnings appeared this day the first time. Up to now every day some warnings less then 10 appeared and I "ignored" them. After I restarted Loxone the warnings appear again
![image](https://github.com/UncleSamSwiss/ioBroker.loxone/assets/17279799/cd03aa17-1c63-4cde-a067-248e9a847fbf)

It seams that I can change the value manually.

**Comment by macpo** [comment]
It seams that all state changes initiated by javascript are delayed. If I update the same value manually no warning appears.
Between the two changes I set the values manually without warning.
![image](https://github.com/UncleSamSwiss/ioBroker.loxone/assets/17279799/c69df241-6fd2-494d-abaa-d34f7dc7394a)

**Comment by macpo** [comment]
If I can test, do, log, jump or anything else to help just drop me a message...

**Comment by raintonr** [comment]
> It seams that all state changes initiated by javascript are delayed.

That seems super weird. The Loxone adapter doesn't care what causes the update. I would suspect your JS is making multiple very fast updates or similar.

**Comment by macpo** [comment]
I have strange problems which could be caused by duplicated sets of values. Perhaps it existed a long time and the new feature #399 shows the behaviour.

**Comment by macpo** [comment]
> That seems super weird. The Loxone adapter doesn't care what causes the update. I would suspect your JS is making multiple very fast updates or similar.
Do you have an idea how I can check it? I tested to change a separate value with a single setState in a sinlge script, but the warning does not appear at the moment...

**Comment by macpo** [comment]
It seems that my setup with two javascript instances (one for important things like switching light and one for monitoring) could be the reason. Today I got two log messages although it should be executed only once.
If the issue appears again running only one instance I will create a new ticket.
